config.json

```diff
@@ -378,6 +378,17 @@
         "random"
       ]
     },
+    {
+      "slug": "yacht",
+      "uuid": "8e129e54-6b5c-4696-84bc-588bc1c7f0f7",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 3,
```

Comment:
Do you think this an appropriate difficulty?

Reply:
Yes, I think it's fine.